### PR TITLE
Update the JVM-SC to work with python3 and a few other fixes

### PR DESCRIPTION
**Pull request by agunde406**
To test, look at the updated sdk/README.md JVM-SC Intkey Transaction Processors section and follow the steps.

### CODE REVIEW

File: sdk/examples/intkey_jvm_sc/sawtooth_intkey/cli/generate.py [hunk 1]
```diff
@@ -108,25 +109,28 @@ def create_batch(transactions, private_key, public_key):
...
-        signature=signature)
+        header_signature=signature)
 
     return batch
 
 
 def get_address(namespace, data):
-    jvm_sc_prefix = hashlib.sha512(namespace).hexdigest()[0:6]
-    addr = jvm_sc_prefix + hashlib.sha512(data).hexdigest()
+    jvm_sc_prefix = hashlib.sha512(namespace.encode("utf-8")).hexdigest()[0:6]
```
Comment (peterschwarz) [inline sdk/examples/intkey_jvm_sc/sawtooth_intkey/cli/generate.py]: Don't need the `"utf-8"` it's the default.

**Comment by delventhalz** [comment]
Perhaps this is a change for a different PR, but do we want intkey to use the Rest Api rather than Stream?

**Comment by agunde406** [comment]
I already have a story to change intkey to use the Rest api

**Comment by delventhalz** [comment]
Should `tp_intkey_jvm_sc` default to verbose mode? It is _very_ verbose at the moment.

**Comment by agunde406** [comment]
We currently do not have a configurable verbose logging setting in the java sdk,

**Pull request merged**